orb/orb.yml

```diff
@@ -221,9 +243,97 @@ commands:
               $SUDO unzip -o ./vault_$VERSION\_linux_amd64.zip -d /usr/bin/. && \
               $SUDO chmod +x /usr/bin/./vault
               $SUDO rm ./vault_$VERSION\_linux_amd64.zip; }
-            
+
+  vault-login:
+    steps:
+      - vault_install
+      - run:
+          name: Secret-injector - Vault login
+          command: |
+            source $BASH_ENV || true
+
             vault login -no-print -method=userpass username=$VAULT_USERNAME password=$VAULT_PASSWORD
             echo 'export VAULT_TOKEN=$(cat $HOME/.vault-token)' >> $BASH_ENV
+
+  python_install:
+    steps:
+      - run:
+          name: Install Python
+          command: |
+
+            ( command -v python3 && command -v pip ) >/dev/null 2>&1 || { apk add --no-cache python3 py3-pip; }
```

Comment:
Are you sure we always can rely on it to be run on a Alpine based image ?

Reply:
Hmmm... come to think of properly not I'll copy paste the parts from cci-common I left out 😄